src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -126,8 +136,8 @@ private boolean isSameSession(AudioFileInfo groupedAudioFileInfo, AudioFileInfo
             && groupedAudioFileInfo.getEndTime().equals(ungroupedAudioFileInfo.getEndTime());
 
         boolean hasContinuity = ungroupedAudioFileInfo.getChannel().equals(groupedAudioFileInfo.getChannel())
-            && (timeOverlaps(ungroupedAudioFileInfo, groupedAudioFileInfo.getEndTime().plusSeconds(acceptableAudioGapSecs))
-            || timeOverlaps(groupedAudioFileInfo, ungroupedAudioFileInfo.getEndTime().plusSeconds(acceptableAudioGapSecs)));
+            && (timeOverlaps(ungroupedAudioFileInfo, groupedAudioFileInfo.getEndTime().plusSeconds(allowableAudioGap.toSeconds()))
+            || timeOverlaps(groupedAudioFileInfo, ungroupedAudioFileInfo.getEndTime().plusSeconds(allowableAudioGap.toSeconds())));
```

Comment:
Since `Duration` implements `TemporalAmount` I think this:

```java
.plusSeconds(allowableAudioGap.toSeconds())
```

can be simplified to:

```java
.plus(allowableAudioGap)
```

Reply:
Simplified as per recommendation